suites/pacific/rbd/tier-2_rbd_encryption.yaml

```diff
@@ -0,0 +1,103 @@
+# Tier2: RBD luks encryption testing
+#
+# This test suite runs test scripts to evaluate the functionality of
+# luks encryption in RBD.
+#
+# Conf File - conf/pacific/rbd/tier-0_rbd.yaml
+#
+# The following tests are covered
+#  - CEPH-83575263 - Encrypt & decrypt file using same keys and different keys
+#  - CEPH-83575251 - Encrypt image and clone using combinations of encryption type
+
+tests:
+
+   #Setup the cluster
+
+  - test:
+      abort-on-fail: true
+      module: install_prereq.py
+      name: install ceph pre-requisites
```

Comment:
Please add single line spacing for each test

Reply:
Done.